### PR TITLE
charts(pod-memory-hog): Add Chaos Chart for pod memory hog experiment

### PR DESCRIPTION
**Pull request by uditgaurav**
- Adding chaos chart for pod memory hog experiment. 
This include addition of:
  - engine
  - rabc
  - chart
  - csv file
  - experiment icon
issue - https://github.com/litmuschaos/litmus/issues/1414
Signed-off-by: Udit Gaurav <uditgaurav@gmail.com>

### CODE REVIEW

File: charts/generic/pod-memory-hog/pod-memory-hog.chartserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    - Causes high Memory resource consumption using dd command which maintains a temporary buffer in memory to write the chunk of data assigned in bs of the dd command.
+    - The application pod should be healthy once chaos is stopped. Expectation is that service-requests should be served despite chaos.
+  keywords:
+    - Kubernetes
+    - Memory
+  platforms:
+    - GKE
+    - Packet(Kubeadm)
+    - Minikube
+    - EKS
```
Comment (ispeakc0de) [inline charts/generic/pod-memory-hog/pod-memory-hog.chartserviceversion.yaml:24]: Can you please verify for EKS?

**Review by ispeakc0de — CHANGES_REQUESTED** [review]

**Comment by ispeakc0de** [comment]
@uditgaurav  Can you please add this experiment to package.yaml and add a .png file too

**Review by ksatchit — APPROVED** [review]

File: charts/generic/pod-memory-hog/pod-memory-hog.chartserviceversion.yaml (hunk 1 above)
Comment (uditgaurav) [reply charts/generic/pod-memory-hog/pod-memory-hog.chartserviceversion.yaml:24]: Its running fine in EKS also @ispeakc0de

Comment (ispeakc0de) [reply charts/generic/pod-memory-hog/pod-memory-hog.chartserviceversion.yaml:24]: great

**Review by ispeakc0de — APPROVED** [review]

**Pull request merged**